pyladies_avatars_rotation.csv

```diff
@@ -1,16 +1,42 @@
+Brazil,Fortaleza,https://hugovk.github.io/python-logos/pyladies/img/PyLadies Fortaleza.png,https://www.facebook.com/PyLadiesFortaleza/,pyladiesfort
+Brazil,Goiânia,https://hugovk.github.io/python-logos/pyladies/img/pyladies-goiania.png,https://www.instagram.com/pyladies_goiania/
+Brazil,Maceió,https://hugovk.github.io/python-logos/pyladies/img/PyLadiesMaceio.png,https://www.facebook.com/PyLadiesMaceio/,pyladiesmaceio
+Brazil,Manaus,https://hugovk.github.io/python-logos/pyladies/img/pyladiesmanaus.png,https://www.facebook.com/pyladiesmanaus/,pyladiesmanaus
+Brazil,Natal,https://hugovk.github.io/python-logos/pyladies/img/pyladies_natal.svg,https://pyladiesnatal.github.io/,PyladiesNatal
+Brazil,Paraíba,https://hugovk.github.io/python-logos/pyladies/img/pyladiespb.png,https://www.facebook.com/pyladiespb,pyladiespb
+Brazil,Parnaíba,https://hugovk.github.io/python-logos/pyladies/img/pyLadiesPHB.png,https://pyladiesparnaiba.carrd.co/,pyladiesphb
+Brazil,Porto Alegre,https://hugovk.github.io/python-logos/pyladies/img/pyladies-porto-alegre.png,https://pyladiespoa.pythonanywhere.com/,Pyladiespoa
+Brazil,Recife,https://hugovk.github.io/python-logos/pyladies/img/pyladiesrecife.png,https://linktr.ee/pyladiesrecife,pyladies_recife
+Brazil,Ribeirão Preto,https://hugovk.github.io/python-logos/pyladies/img/Pyladies-RP.png,https://pyladies-rp.github.io/,pyladies_recife
+Brazil,Salvador,https://hugovk.github.io/python-logos/pyladies/img/pyladies-salvador.png,https://pyladiessalvador.github.io/,pyladiessa
+Brazil,Salvador,https://hugovk.github.io/python-logos/pyladies/img/pyladiessalvador.png,https://pyladiessalvador.github.io/,pyladiessa
+Brazil,Sergipe,https://hugovk.github.io/python-logos/pyladies/img/pyladies_sergipe.svg,https://www.instagram.com/pyladiessergipe/
+Brazil,Sorocaba,https://hugovk.github.io/python-logos/pyladies/img/PyLadies Sorocaba.png,https://www.facebook.com/PyLadiesSorocaba,pyladiesorocaba
+Brazil,Sul de Minas Gerais,https://hugovk.github.io/python-logos/pyladies/img/pyladies_sul_de_minas.png,https://www.facebook.com/pyladiesmg/
+Brazil,São Carlos,https://hugovk.github.io/python-logos/pyladies/img/pyladies-saocarlos.png,https://www.facebook.com/PyLadiesSaoCarlos,PyLadiesSanca
+Brazil,São José dos Campos,https://hugovk.github.io/python-logos/pyladies/img/PyLadies São José dos Campos.png,https://www.facebook.com/pyladiesSJC/
+Brazil,São Luís,https://hugovk.github.io/python-logos/pyladies/img/pyladies-sao-luis.svg,https://www.instagram.com/pyladiesslz/,pyladiesslz
+Brazil,Teresina,https://hugovk.github.io/python-logos/pyladies/img/pyladiesteresina.png,https://pyladiesteresina.github.io/,pyladiesthe
+Brazil,Vale Do Paraiba,https://hugovk.github.io/python-logos/pyladies/img/pyladiesvale.png,http://vale.pyladies.com/,pyladiesvale
+Global,Remote,https://hugovk.github.io/python-logos/pyladies/img/remote-logo.svg,https://remote.pyladies.com/,PyLadiesRemote
```

Comment:
```suggestion
Remote,Remote,https://hugovk.github.io/python-logos/pyladies/img/remote-logo.svg,https://remote.pyladies.com/,PyLadiesRemote
```

We started using "Global" PyLadies as the main PyLadies org, while the Remote PyLadies is still considered a "local" or "over the internet" group. So I think it's best to separate the meaning of "Global" from the "Remote" chapter.